bundles/org.eclipse.equinox.launcher.cocoa.macosx.aarch64/build.properties

```diff
@@ -14,10 +14,8 @@
 bin.includes = META-INF/,\
                launcher.cocoa.macosx.aarch64.properties,\
                about.html
-               
-customBuildCallbacks=customBuildCallbacks.xml
+
 generateSourceBundle=false
-binaryTag=v20221103-0506
```

Comment:
Since the we are removing binaryTag=v20221103-0506 from the build.properties. please remove the code to update it. The code is located at https://github.com/Torbjorn-Svensson/eclipse-equinox_equinox/blob/b1191e227e22399d7ff70f14e48b8b89a8725802/releng/org.eclipse.equinox.launcher.releng/build.xml#L246

Reply:
Decided to keep it in build.properties.